docker/pypi/wmagent-base/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN apt-get update
 RUN apt-get install -y libmariadb-dev-compat libmariadb-dev apache2-utils cron mariadb-server myproxy voms-clients voms-clients-java rlwrap libaio1 && apt-get clean
 
 # Install some debugging tools
-RUN apt-get install -y hostname net-tools iputils-ping procps && apt-get clean
+RUN apt-get install -y hostname net-tools iputils-ping procps jq && apt-get clean
```

Comment:
I feel like it would be even better to add this dependency to the `dmwm-base` image, but that requires then much more effort to update everything else. So I think it's fine to keep it here, eventually when we update dmwm-base, we can add it there and remove it from here.

Reply:
As you wish, I am fine either way. I put it here because it is not "necessary", it is a "nice to have" and felt consistent with these "debugging tools". but as you wish, i can move it to dmwm-base image and rebuild other images if you prefer this way :)